DataFormats/L1TParticleFlow/interface/egamma.h

```diff
@@ -87,15 +87,16 @@ namespace l1ct {
     z0_t hwZ0;
     tkdeta_t hwDEta;  // relative to the region center, at calo
     tkdphi_t hwDPhi;  // relative to the region center, at calo
+    id_score_t hwIDScore;
```

Comment:
BTW the two comments above on hwDEta and hwDPhi are misleading

Reply:
removed